### ISSUE TITLE
under dispatch/Work Assignments the elements Hourly, Daily, Weekly and Monthly should be coded as tabs, and read as tabs by AT

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12138]
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role

On the view Dashboard page under dispatch/Work Assignments the elements Hourly, Daily, Weekly and Monthly are visually stylized as headings, but are read as links for assistive technology users. Functionally these are tabs and should be coded and read as tabs for assistive technology users. (New: 08/27/2019)

**Comment by leikkisa** [comment]
It turns out this is for the dispatch stats page, which has pretty old code and is also using a ruby view instead of our regular client code.  @Sjones352 Looked in Google Analytics whether anyone was visiting the dispatch stats page, and we saw 0 views since October 1st, for both the dispatch and certification stats pages.  Intake stats was deprecated, so we decided instead of fixing this accessibility ticket, it makes more sense to deprecate the stats pages altogether.

Closing this ticket in favor of: #12991
